### ISSUE TITLE
RC Test: Friction 1.4.0-rc.4 for Keyboard Navigation 

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#78]
previous rc test: https://github.com/phetsims/QA/issues/75
@emily-phet @terracoda @jessegreenberg, Friction 1.4.0-rc.4 is ready for RC testing.

**[Link to sim](https://www.colorado.edu/physics/phet/dev/html/friction/1.4.0-rc.4/friction_en.html)**

**[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/friction/1.4.0-rc.4/friction_en-iframe.html)**

**[Testing Matrix](https://docs.google.com/spreadsheets/d/1cnoPcw-DPCOpy4fTgoDPQhb0_sX2Z26J3W66Z3IQxtw/edit#gid=2055394226)**

**Issues to Verify**
Please test the following issues and check them off after addressing. If they are resolved, close them.  If not, they should be updated.

- [x] https://github.com/phetsims/friction/issues/57
- [x] https://github.com/phetsims/friction/issues/54
- [x] https://github.com/phetsims/friction/issues/53
- [x] https://github.com/phetsims/friction/issues/52

If any new issues are found, please note them in https://github.com/phetsims/friction/issues and reference this issue.  

@ariel-phet please assign and prioritize

**Comment by jessegreenberg** [comment]
@phet-steele pointed out that the  version id still read as 1.4.0-rc.2 in the about dialog, so we wanted to make sure that the correct build is being tested. Looking at the order of commits, it looks like the build steps might have been
- grunt
- bump package.json
- grunt deploy-rc

And build-server uses the dependencies.json in the build directory so it would miss the update to version. Seems safe, but we will build a new RC to correct the version number.

**Comment by jessegreenberg** [comment]
Re-built and updated version id to rc.4. Thanks @phet-steele, back to you.

**Comment by phet-steele** [comment]
Ready to be published

**Comment by terracoda** [comment]
Woo! Hoo!

**Comment by jessegreenberg** [comment]
Sim deployed, thank you @phet-steele and team!

https://phet.colorado.edu/sims/html/friction/latest/friction_en.html
